build/int.cloudbuild.yaml

```diff
@@ -33,9 +33,49 @@ steps:
 - id: destroy
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do destroy']
+- id: go-init-logbucket-folder
+  waitFor:
+    - prepare
+  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  args: ['/bin/bash', '-c', 'cd test/integration && RUN_STAGE=init go test -v -run TestLogBucketFolderModule ./... -p 1']
```

Comment:
We should switch to CLI instead
example https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/build/int.cloudbuild.yaml

Reply:
ack